specification/Metadata/Semantics/README.md

```diff
@@ -0,0 +1,73 @@
+# Cesium Metadata Semantic Reference
+
+This document defines a general set of semantics for 3D Tiles and glTF. Tileset authors may define their own application- or domain-specific semantics separately.
+
+Semantics describe how properties should be interpreted. For example, an application that sees the `HORIZON_OCCLUSION_POINT` semantic would use the property for horizon occlusion culling as defined below.
+
+```jsonc
+{
+  "properties": {
+    "horizonOcclusionPoint": {
+      "type": "ARRAY",
+      "componentType": "FLOAT64",
+      "componentCount": 4,
+      "semantic": "HORIZON_OCCLUSION_POINT",
+    },
+    "name": {
+      "type": "STRING",
+      "semantic": "NAME"
+    }
+  }
+}
+```
+
+For full usage see:
```

Comment:
I, the novice reader, don't yet understand why this spec (is it a spec?) is sending me to another spec "for full usage" - nor do I yet understand what usage is defined here or there.

I imagine this spec basically defines the vocab and maybe the runtime/application behavior.

Then these "full usage" specs define the encoding, how it attaches to concepts in those specs, and maybe the runtime/application behavior or at least the interaction of the behavior defined here in the context of the other spec.

The reader needs a roadmap to know what to expect where.

And a diagram.

Reply:
Ah, I see that this doesn't really define any behavior but kinda tries to some time.

The behavior line needs to be crystal clear.